### PR TITLE
[feat] : SDE backend : Refactor DT registry use through EDC connector : rel-2.3.0

### PR DESCRIPTION
**Pull request by ChetanT-System**
### Added
- DT use refactor for look up twin
## Pre-review checks
Please ensure to do as many of the following checks as possible, before asking for committer review:
- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by ChetanT-System** [comment]
Hi @dvasunin, @adityagajbhiye9 Please review the PR and approve.

File: modules/sde-external-services/digital-twins/src/main/java/org/eclipse/tractusx/sde/digitaltwins/facilitator/DigitalTwinsFacilitator.java [hunk 1]
```diff
@@ -92,44 +83,46 @@ public List<String> shellLookupFromDDTR(ShellLookupRequest request, String ddtrU
...
 		} catch (Exception e) {
-			String error = "Error in deleteShell :" + digitalTwinsHost + ", " + shellId + "," + e.getMessage();
+			String error = "Error in deleteShell : " + shellId + "," + e.getMessage();
 			log.error(error);
 			throw new ServiceException(error);
 		}
 		return deleteResponse;
 	}
 
 	@SneakyThrows
```
Comment (dvasunin) [inline modules/sde-external-services/digital-twins/src/main/java/org/eclipse/tractusx/sde/digitaltwins/facilitator/DigitalTwinsFacilitator.java:104]: as far as I see @SneakyThrows is redundant here

File: modules/sde-external-services/edc/src/main/java/org/eclipse/tractusx/sde/edc/services/ConsumerControlPanelService.java [hunk 2]
```diff
@@ -234,4 +248,114 @@ public void subscribeDataOffers(ConsumerRequest consumerRequest, String processI
...
+			throw new ServiceException(errorMsg);
+		} catch (InterruptedException ie) {
+			Thread.currentThread().interrupt();
+			String errorMsg = "InterruptedException for asset " + assetId + "," + ie.getMessage();
+			log.error(errorMsg);
+			throw new ServiceException(errorMsg);
+		} catch (Exception e) {
+			String errorMsg = "Exception for asset " + assetId + "," + e.getMessage();
+			log.error(errorMsg);
+			throw new ServiceException(errorMsg);
```
Comment (dvasunin) [inline modules/sde-external-services/edc/src/main/java/org/eclipse/tractusx/sde/edc/services/ConsumerControlPanelService.java:310]: consider to make ServiceException be a subclass of RuntimeException. There are many cases where methods which throw  ServiceException are annotated with @SneakyThrows

**Review by dvasunin — APPROVED** [review]
there are minor comments with suggestions about code improvement which may be taken into account in the future. For now LGTM

**Review by almadigabor — APPROVED** [review]
LGTM

**Comment by ChetanT-System** [comment]
We have Fixed issue #70 already. Thank you @almadigabor !

**Pull request merged**